testnet/stacks-node/src/burnchains/bitcoin_regtest_controller.rs

```diff
@@ -658,36 +660,65 @@ impl BitcoinRegtestController {
     }
 
     fn finalize_tx(
-        &self,
+        &mut self,
         tx: &mut Transaction,
         total_spent: u64,
-        utxos: Vec<UTXO>,
+        mut utxos: Vec<UTXO>,
         signer: &mut BurnchainOpSigner,
+        attempt: u64,
     ) -> Option<()> {
-        let tx_fee = self.config.burnchain.burnchain_op_tx_fee;
+        // spend UTXOs in decreasing order
+        utxos.sort_by(|u1, u2| u1.amount.cmp(&u2.amount));
+        utxos.reverse();
+
+        // RBF
+        let tx_fee = self.config.burnchain.burnchain_op_tx_fee
+            + (attempt * self.last_tx_len * self.min_relay_fee);
```

Comment:
Should we be less aggressive on the bid for the RBF? Could it be + 1 sat per attempt?

Reply:
Nope, it has to be higher.  Per BIP125:

> The replacement transaction must also pay for its own bandwidth at or above the rate set by the node's minimum relay fee setting. For example, if the minimum relay fee is 1 satoshi/byte and the replacement transaction is 500 bytes total, then the replacement must pay a fee at least 500 satoshis higher than the sum of the originals.